cli/cmd/agent_list.go

```diff
@@ -175,15 +175,15 @@ func listAgents(_ *cobra.Command, _ []string) error {
 
 	cli.OutputHuman(
 		renderSimpleTable(
-			[]string{"MID", "Short Agent Token", "Hostname", "Name", "IP Address", "External IP", "OS Arch", "Last Checkin"},
-			machineDetailsToListAgentTable(machines),
+			[]string{"MID", "Short Agent Token", "Hostname", "Name", "Status", "IP Address", "External IP", "OS Arch", "Last Checkin"},
```

Comment:
Add column created time and use the `lastUpdate` field for the Last Checkin column.

Order table like:
```suggestion
			[]string{"MID", "Short Agent Token", "Status", "Hostname", "Name", "IP Address", "External IP", "OS Arch", "Last Check-in", "Created Time"},
```

Choose if Created at or Created time